### PR TITLE
Created Input component

### PR DESCRIPTION
**Pull request by pritam-kr**
Created Input component  of different types and with two form

- signup form
- signIn form
- input with text
- input with number
- input with  password
- error styling
- input checkbox
- input radio button
- text area input
- disabled input
- secondary input

### CODE REVIEW

**Review by ScoobyDubeyDo — APPROVED** [review]

File: components/Input.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    width: 5%;
+    font-size: 3rem;
+}
+
+.form {
+    background-color: var(--body-background);
+    border: 1px solid var(--border-color);
+    width: 350px;
+    border-radius: 5px;
+    padding: 1.5em;
```
Comment (himadri2110) [inline components/Input.css:54]: You can use CSS variable for `border-radius` to keep it consistent throughout. 

Rest is all good to go!

**Review by himadri2110 — APPROVED** [review]

File: components/Input.css (hunk 1 above)
Comment (pritam-kr) [reply components/Input.css:54]: I'll fix in refactor branch Thank you

**Review by AbhishekkGautam — COMMENTED** [review]
LGTM

**Comment by preethasri** [comment]
Looks good 👍

**Pull request merged**